pkg/asset/installconfig/ovirt/ovirt.go

```diff
@@ -37,6 +37,11 @@ func Platform() (*ovirt.Platform, error) {
 		return nil, err
 	}
 
+	err = engineMinimumVersionRequirement(c)
```

Comment:
it is incorrect to add this validation to install config terminal generation as that doesn't help users that are using customized install-config. I think you should look at https://github.com/openshift/installer/blob/45e9101b2e05fc6fb24d209a744569b1b36a49b6/pkg/asset/installconfig/platformprovisioncheck.go

Reply:
Thanks, going to check how this work and update the PR.